Makefile

```diff
@@ -536,7 +536,7 @@ validate.completions: completions/bash/podman
 	. completions/bash/podman
 	if [ -x /bin/zsh ]; then /bin/zsh completions/zsh/_podman; fi
 
-validate: gofmt .gitvalidation validate.completions golangci-lint man-page-check
+validate: lint gofmt .gitvalidation validate.completions golangci-lint man-page-check
```

Comment:
This is probably not going to work, because the `lint` target requires `pre-commit` which may not exist on all testing environments (this was deliberate as part of #4809).

And please, if you do solve the dependency issue, do not add duplication here: you should not have both `lint` and `golangci-lint` on this line.